bc_obps/registration/api/business_structure.py

```diff
@@ -7,7 +7,7 @@
 
 ##### GET #####
 @router.get("/business_structures", response=List[BusinessStructureOut])
-@authorize(AppRole.get_all_authorized_roles())
+@authorize(AppRole.get_all_authorized_app_roles(), AppRole.get_all_industry_user_operator_roles())
```

Comment:
Not sure what you mean by this--a separate method to return both arguments? I think I'd like to keep two arguments so it's obvious that there are two places to look for permissions. Thoughts?

Reply:
Could be something like this:
```python
def func_name():
    authorized_app_roles = AppRole.get_all_authorized_app_roles()
    industry_user_operator_roles = AppRole.get_all_industry_user_operator_roles()
    return authorized_app_roles, industry_user_operator_roles

@authorize(*get_authorization_roles())
def your_function_here():
   ...
```
If you think this would be more confusing, skip this suggestion :)

Reply:
Ah, gotcha! I can't think of a really clear name for `func_name` at the moment, so I think I'll leave it as two separate arguments. If I or anyone has a brainwave later we can update then!